### ISSUE TITLE
Extension interface has array type for value

### ISSUE THREAD

**Issue by jpbow** [octoshikari/aidbox-tool#173]
**Describe the bug**
Extension property `value` should be singular, not an array (at least for FHIR)

```ts
/* Base StructureDefinition for Extension Type: Optional Extension Element - found in all resources. */
export interface Extension extends Element {
  /* identifies the meaning of the extension */
  url: uri;
  /* Value of extension */
  value?: {
    string?: string;
    uri?: uri;
    // ...
    DataRequirement?: DataRequirement;
  }[];  // <-- Should not be an array
}
```

**Comment by octoshikari** [comment]
Hi @jpbow. Will fix it and let you know

**Comment by octoshikari** [comment]
@jpbow  Hi! fixed in v0.4.6
